extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -559,7 +559,7 @@ private static boolean shouldFireOidcServerAvailableEvent(String tenantId) {
     }
 
     private static boolean fireOidcServerEvent(String authServerUrl, SecurityEvent.Type eventType) {
-        if (Arc.container().instance(SecurityConfig.class).get().events().enabled()) {
+        if (ConfigProvider.getConfig().getOptionalValue("quarkus.security.events.enabled", boolean.class).orElse(true)) {
```

Comment:
@michalvavrik Sure, lets only have a constant for it

Reply:
it's only used once, but if that's what you prefer I'll add it. Where do you want to add it, OIDC recorder is enough?

Reply:
Added constant to the recorder.